src/Orleans/Configuration/ClientConfiguration.cs

```diff
@@ -329,7 +331,7 @@ internal void LoadFromXml(XmlElement root)
                             }
                             break;
                         case "Telemetry":
-                            ConfigUtilities.ParseTelemetry(child);
+                            ConfigUtilities.ParseTelemetry(child, this.MetricsConfiguration);
```

Comment:
I thought we are moving away from xml config in 2.0. What's the thinking behind this ?